apps/sparo-lib/src/cli/commands/checkout.ts

```diff
@@ -108,6 +115,10 @@ export class CheckoutCommand implements ICommand<ICheckoutCommandOptions> {
         targetProfileNames = profiles;
       }
 
+      if (addProfiles.length) {
+        targetProfileNames.push(...addProfiles);
```

Comment:
I have a strong feeling now that the "targetProfileNames" should be a Set type to deduplicate same profile name.

Reply:
Good suggestion, I have changed all profiles' type to be Set<string>